feature/settings/push/src/main/kotlin/app/k9mail/feature/settings/push/ui/PushFoldersActivity.kt

```diff
@@ -0,0 +1,62 @@
+package app.k9mail.feature.settings.push.ui
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.result.contract.ActivityResultContract
+import app.k9mail.core.ui.compose.theme.K9Theme
+import com.fsck.k9.Account.FolderMode
+
+/**
+ * Screen to change the "Push folders" setting of an account.
+ */
+class PushFoldersActivity : ComponentActivity() {
```

Comment:
We should not use Activites for features, this renders the move to Jetpack Compose harder. There is a dependency on `:app:core` which makes everything complicated. The future permission management should be isolated and moved to a `core:permission`module.

Reply:
There's no easy way to have a `androidx.preference.Preference` that opens a separate screen. Using a separate Activity required the least amount of changes. Given that this is a very short-lived feature I felt it wasn't worth spending more time on this than necessary.